string-manipulation-2 - Tré Mitchell.

TreMitchell opened the pull request:
<img width="1440" alt="Screenshot 2024-09-02 at 11 14 34 PM" src="https://github.com/user-attachments/assets/f356da61-546c-4583-9e71-68dc718f5339">

TreMitchell committed:
Completed string-manipulation-2 exercise.

shawnkost commented on string-manipulation-2/src/capitalize-word.ts:
```diff
@@ -0,0 +1,13 @@
+/* exported capitalizeWord */
+function capitalizeWord(word: string): string {
+  if (word === 'jaVAsCrIPt' || word === 'javaScript' || word === 'JavascRipt') {
```
If you convert the `word` to lowercase, you don't need to check for so many variations

shawnkost reviewed (approved):
👍🏻

TreMitchell committed:
Added a lowercase function to the first `if` statement.

Updated the first `if` statement to have a lowercase function instead of different variations of a word.
